stock_picking_auto_create_lot_biochousa/views/partner_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 Solvos Consultoría Informática - Christian Santamaría
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_partner_property_form" model="ir.ui.view">
```

Comment:
In general, there's a problem using this view: this section is only shown if debug mode is activated, you can check it. "Inventory" section apparently was the more suitable one, but we should look for an alternative (e.g. an independent section within "Sales & Purchase" tab)